bot/resources/tags/message-content-intent.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: ["mcintent", "message_content", "message_content_intent"]
+embed:
+    title: "Discord Message Content Intent"
+---
+
+The Discord gateway only dispatches events you subscribe to, which you can configure by using "intents."
+
+The message content intent is what determines if an app will receive the actual content of newly created messages. Without this intent, discord.py won't be able to detect prefix commands, so prefix commands won't respond.
+
+Privileged intents, such as message content, have to be explicitly enabled from the [Discord Developer Portal](https://discord.com/developers/applications) in addition to being enabled in the code:
+
+```py
+intents = discord.Intents.default() # create a default Intents instance
+intents.message_content = True # enable message content intents
+
+bot = commands.Bot(command_prefix="!", intents=intents) # actually pass it into the constructor
+```
```

Comment:
I think mentioning other relevant tags at the end would be helpful
````suggestion
```
For more information on intents, see `/tag intents`. If prefix commands are still not working, see `/tag on-message-event`.
````

Reply:
Speaking of the intents tag, I've just noticed there is a lot of overlap between it and this, maybe we could differentiate the two a bit more?

Reply:
Thats a good point. I think the beginning is important to have, but maybe removing the part about the privilaged intents, and linking it would be useful. Also, incorporating the fact that you do have to enable it in the discord developer portal at the beginning. Does that sound good?

Reply:
That sounds good to me. I think the intents tag can also be improved (primarily the length because it is very wordy at the moment) but it would be better addressed in a separate PR.